src/js/components/NasaTable.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TablePagination from '@material-ui/core/TablePagination';
+import ImageDialog from './ImageDialog';
+import { getNasaData } from '../api/table'
+
+class NasaTable extends Component {
+
+    constructor (props) {
+        super(props)
+        this.state = {
+            data: {},
+            visibleRows: null,
+            page: 0,
+            rowsPerPage: 10,
+            open: false,
```

Comment:
Call it more declarative, related to its state like isModalOpened or isModalVisible etc

Reply:
done